lib/core/src/Cardano/Wallet/Primitive/Slotting.hs

```diff
@@ -611,3 +634,54 @@ unsafeExtendSafeZone = f . neverFails r
         , handleResult = h
         }
     r = "unsafeExtendSafeZone should make PastHorizonExceptions impossible."
+
+
+{-------------------------------------------------------------------------------
+                        Cardano API EraHistory
+-------------------------------------------------------------------------------}
+
+dummyEraHistory :: (Cardano.EraHistory Cardano.CardanoMode, SystemStart)
+dummyEraHistory =
```

Comment:
Having to use this for testing and having to add a separate `eraHistory` to `NetworkLayer` — I'm not happy about it. It does work, however, and might be preferable to the alternatives...